packages/frontend/src/build/buildPages.ts

```diff
@@ -23,7 +24,9 @@ async function main() {
 
   const http = new JsonHttpClient(new HttpClient(), config.backend.skipCache)
 
-  const tvlApiResponse = await fetchTvlApi(config.backend.apiUrl, http)
+  const tvlApiResponse = config.features.detailedTvl
```

Comment:
This one is risky, the endpoint is also behind the feature flag on the backend, so after you enable it here and forget to do so on the backend, the build will fail. I thought that the detailed TVL would be possibly undefined and fetched in a different step, and later passed as a fourth argument to createApi() wdyt?